apps/versions/tests.py

```diff
@@ -833,6 +840,22 @@ def test_success(self):
         assert self.addon.current_version
         self.assert_served_by_mirror(self.client.get(self.latest_url))
 
+    def test_beta(self):
+        r = self.client.get(self.latest_beta_url)
+        eq_(r.status_code, 302)
+        beta_file_url = reverse('downloads.file', args=[self.beta_file.id])
+        url = beta_file_url + '/' + self.beta_file.filename
+        assert r['Location'].endswith(url), r['Location']
+
+        # No beta downloads for unreviewed addons
+        self.addon.status = amo.STATUS_PENDING
+        self.addon.save()
+        r = self.client.get(self.latest_beta_url)
+        eq_(r.status_code, 302)
+        beta_file_url = reverse('downloads.file', args=[self.beta_file.id])
+        url = self.file_url + '/' + self.file.filename
+        assert r['Location'].endswith(url), r['Location']
+
```

Comment:
I would love a test here for the case where the addon is public, but doesn't have a beta file (and thus returns a 404? From the current code, I believe it'll return the latest version, but I'm not sure that's what we want, see my comment below).